lib/src/main/java/com/diffplug/spotless/generic/LicenseHeaderStep.java

```diff
@@ -166,18 +167,29 @@ private Runtime(String licenseHeader, String delimiter, String yearSeparator, bo
 
 			Optional<String> yearToken = getYearToken(licenseHeader);
 			if (yearToken.isPresent()) {
-				yearToday = String.valueOf(YearMonth.now().getYear());
+				this.yearToday = String.valueOf(YearMonth.now().getYear());
 				int yearTokenIndex = licenseHeader.indexOf(yearToken.get());
-				beforeYear = licenseHeader.substring(0, yearTokenIndex);
-				afterYear = licenseHeader.substring(yearTokenIndex + yearToken.get().length());
-				yearSepOrFull = yearSeparator;
+				this.beforeYear = licenseHeader.substring(0, yearTokenIndex);
+				this.afterYear = licenseHeader.substring(yearTokenIndex + yearToken.get().length());
+				this.yearSepOrFull = yearSeparator;
 				this.updateYearWithLatest = updateYearWithLatest;
+
+				boolean hasHeaderWithRange = false;
+				// year plus separator
+				int lastChars = 4 + yearSeparator.length();
```

Comment:
rename `lastChars` to `yearPlusSep` and delete comment

Reply:
sure, no problem